src/components/HURUmap/Panel/PanelItem.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import RichData from "./RichData";
+
+function PanelItem({ item, ...props }) {
+  const key = item.value;
+  switch (key) {
+    case "rich-data":
+      return <RichData {...props} />;
+
+    default:
```

Comment:
```suggestion
  switch (item.key) {
    case "rich-data":
      return <RichData {...props} />;
    default:
```
What am I missing?

Reply:
so this was before I realized the other places had been disabled, but the idea was to be able to support the other panels as wazimap, currently, the design only supports Rich Data, ~Ponit Mapper~, ~Data Mapper~  @kilemensi

Reply:
I figured out that part @KobbyMmo ... I meant why `const key = item.value;` for `key` to only be used on `switch(key)`? Why not just use `switch(item.value)` directly if we're not using `key` in multiple places?